automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/azure/AzureRuntimeSpec.scala

```diff
@@ -76,10 +75,10 @@ class AzureRuntimeSpec
           s"AzureRuntimeSpec: disk ${workspaceId}/${diskDuringRuntimeCreate.getId()} in creating status detected"
         )
 
-        // Verify the runtime eventually becomes Running
+        // Verify the runtime eventually becomes Running (in 40 minutes)
         monitorCreateResult <- streamUntilDoneOrTimeout(
           callGetRuntime,
-          120,
+          240,
```

Comment:
I did this because it was set to mirror the Leo timeout code, but it is really irrelevant given WSM timeout code is what we should adhere to. The WSM code definitively takes longer than the Leo timeout based on this test run https://console.cloud.google.com/logs/query;cursorTimestamp=2023-08-07T11:13:22.325Z;endTime=2023-08-07T12:11:26.086Z;query=resource.labels.namespace_name%3D%22terra-leonardo-5783806364-1-dev%22%0A--%20resource.labels.container_name%3D%22leonardo-standalone%22%0A--%20severity%3DERROR%0A--%20trace%3D%22projects%2Fbroad-dsde-qa%2Ftraces%2F224c30624837f3276780fc1ddb36bca7%22%0A--%20%22automation-test-ancurrcxz%22%0A--%20%22create-vm-1%22%0Aresource.labels.container_name%3D%22workspacemanager%22%0A%22create-vm-1%22%0Atimestamp%3D%222023-08-07T11:13:22.325Z%22%0AinsertId%3D%22ppdm8y201zxahw8v%22;startTime=2023-08-07T10:22:44.085Z?project=broad-dsde-qa